.github/workflows/tier-2.yaml

```diff
@@ -0,0 +1,46 @@
+---
+name: Tier 2 jobs
```

Comment:
Same here I think we should have a more meaningful title as
`2 - Merge` or `2 - Merge to default branch` rather than `Tier 2 jobs`
Because Tier 2 will always be related to a merge.